build.gradle

```diff
@@ -111,8 +121,8 @@ dependencies {
     testImplementation "org.apache.flink:flink-streaming-java_${scalaBinaryVersion}:${flinkVersion}:tests"
 
     testRuntimeOnly "com.vladsch.flexmark:flexmark-all:0.35.10"
-    testRuntime 'org.pegdown:pegdown:1.4.2'
-    testCompile "info.picocli:picocli:4.2.0"
+    testRuntimeOnly 'org.pegdown:pegdown:1.6.0'
+    testImplementation 'info.picocli:picocli:4.6.1'
```

Comment:
Why did you need to change these scopes? Are those the old dependency categories?

Reply:
Yes, per the [Java library plugin documentation from Gradle](https://docs.gradle.org/3.4/userguide/java_library_plugin.html?_ga=2.49116771.473058149.1632775631-1137966675.1632157466#sec:java_library_configurations_graph), 

> The compile, testCompile, runtime and testRuntime configurations inherited from the Java plugin are still available but are deprecated. You should avoid using them, as they are only kept for backwards compatibility.

`testRuntimeOnly` is supposed to replace `testRuntime`, and `testImplementation` is supposed to replace `testCompile`.  If you try build with the old category names, it should show a warning that also says this.